src/main/resources/templates/search-journeys-results.html

```diff
@@ -47,12 +47,19 @@ <h1 class="govuk-heading-xl ma0" th:inline="text">Manage Journey Price Catalogue
                     </a>
                 </td>
                 <td class="govuk-table__cell" scope="row" th:inline="text">[[${journey.toLocationType}]]</td>
-                <td class="govuk-table__cell" scope="row">
-                    <a th:href="@{'/update-price/' + ${journey.fromNomisAgencyId} + '-' + ${journey.toNomisAgencyId}}"
-                       class="ma0 govuk-link" th:inline="text">
-                        £[[${#numbers.formatDecimal(journey.unitPriceInPounds(), 1, 'COMMA', 2, 'POINT')}]]
-                    </a>
-                </td>
+                <div sec:authorize="!hasRole('ROLE_PECS_MAINTAIN_PRICE')">
```

Comment:
You will see this on the manage journey prices screen if you do not have the maintain prices role (no link to update price):

![image](https://user-images.githubusercontent.com/60104344/113684577-483ee980-96bd-11eb-9c6c-9a766f935d49.png)